README.md

```diff
@@ -1,39 +1,93 @@
+- You can pin your projects by dragging the KRunner results to the taskbar or desktop.
 
-Debian/Ubuntu:  
-`sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libnotify-bin libkf5kcmutils-dev libkf5kio-dev`
+<details>
+<summary><b>Screenshots</b></summary>
+  
+#### Available CLion projects  
+![Available CLion projects](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/multible_projects.png)
+
+#### Search for project by name  
+![Search for project by name](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/launch_by_name.png)
+
+#### Search projects of app  
+![Search for project by name](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/search_projects_of_app.png)
+  
+#### Config Dialog
+![Config Dialog](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/config_dialog.png)
+  
+</details>
+
+If you like this plugin you can check out [the dolphin version](https://github.com/alex1701c/JetBrainsDolphinPlugin) and leave a rating in the [KDE Store](https://www.pling.com/p/1311630/).
+
+## Installation
+### 1 Required Dependencies
+
+<details>
+<summary>Debian/Ubuntu:</summary>
+  
+```
+sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libnotify-bin libkf5kcmutils-dev libkf5kio-dev
+```
+  
+</details>
+
+<details>
+<summary>openSUSE:</summary>
+  
+```
+sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel 
+kservice-devel krunner-devel gettext-tools kconfigwidgets-devel libnotify-tools kcmutils-devel kio-devel
+```
+  
+</details>
+
+<details>
+<summary>Fedora:</summary>
+  
+```
+sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext libnotify kf5-kcmutils-devel kf5-kio-devel qt5-qtquickcontrols2-devel
+```
+  
+</details>
 
-openSUSE:  
-`sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel 
-kservice-devel krunner-devel gettext-tools kconfigwidgets-devel libnotify-tools kcmutils-devel kio-devel`
+<details>
+<summary>Archlinux(Manjaro):</summary>
 
-Fedora:  
-`sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext libnotify kf5-kcmutils-devel kf5-kio-devel`
+```
+sudo pacman -S cmake extra-cmake-modules libnotify kcmutils kio
+```
+  
+</details>
+
+### 2 KRunner Plugin
+#### 2.1 Get Files
 
-Archlinux(Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules libnotify kcmutils kio`
+<details>
+<summary>Option A: Easy oneliner method</summary>
+
+```
+sudo curl https://raw.githubusercontent.com/alex1701c/JetBrainsRunner/master/install.sh | bash
```

Comment:
The install.sh script will ask for admin privileges when installing. The download, configuring and compiling does not require those privileges.